### DISCUSSION TITLE
Support `pylist`, `pydict` etc.

### DISCUSSION THREAD

**Discussion by vkcku** [lk-geimfari/mimesis#1417] [Ideas]
Would it be possible to support creating a list, dictionary, set etc. of random items. For example, in `faker` you could do something like the following:

```python
faker = Faker()

list_items = faker.pylist()
dict_ = faker.pydict()
```

**Comment by vkcku** [comment]
I started this as a discussion because this is not really a difficult thing to manually implement using what `mimesis` already providers.

**Reply by lk-geimfari** [reply]
We can do this easily, but I'm not sure that we should. I mean, it's pretty easy to do this using builtin functionality of Mimesis. 

Although it might be nice to add a separate `Datatype` provider or something to generate different data types.

**Comment by acobb83** [comment]
I'm trying to run mimesis on a PC with no internet access. I install it but get the following issue. Saying its partially initialized. Works fine on a computer with internet access.
![download](https://github.com/lk-geimfari/mimesis/assets/40008673/80a014f7-7a54-489a-93b2-ca561ca8743f)

**Reply by lk-geimfari** [reply]
Hi, can you please show the exact steps to reproduce this?

**Comment by acobb83** [comment]
Hi,

1. Create new virtual environment with Python 3.12
2. Download Poetry Core v1.9.0 and mimesis v17 from pypi.org
3. pip install poetry core
4. pip install mimesis
5. Run environment in Pycharm and VS and get the same error.

**Reply by lk-geimfari** [reply]
Let's continue in #1565.
